index.js

```diff
@@ -31,6 +31,7 @@ async function run() {
             }
             const jobStatus = res.headers.get('location');
             core.info(`Created job: ${jobStatus}`);
+            core.notice(`${b5apisrv}${jobStatus}`);
```

Comment:
Is there a way to specify the API version in the URL?

https://barney-api.infra.corp.arista.io/job/J110ef964089d13ac40f22bd81c7a9a1a907a2c50d1487c410521a0357afef723

this gives
```
{
  "error": "Must specify an API version"
}
```